libqpdf/QPDFObjectHandle.cc

```diff
@@ -2159,6 +2161,7 @@ QPDFObjectHandle::parseInternal(PointerHolder<InputSource> input,
                 object = newDictionary(dict);
                 setObjectDescriptionFromInput(
                     object, context, object_description, input, offset);
+                object.setParsedOffset(offset - 2);
```

Comment:
I'm not sure this handles whitespace properly.

Reply:
If I understand correctly, whitespace is not handled here.
`offset` points to the next character of the beginning of the dictionary "<<".
By `offset - 2`, the parsed offset is changed to point to the "<<".
If you prefer the parsed offset to point the next character of the "<<", I'll fix so.